node/cmd/calico-node/main.go

```diff
@@ -71,6 +72,9 @@ var thresholdTime = flagSet.Duration("threshold-time", 30*time.Second, "Threshol
 var runStatusReporter = flagSet.Bool("status-reporter", false, "Run node status reporter")
 var showStatus = flagSet.Bool("show-status", false, "Print out node status")
 
+// Options for watching node flowlogs.
+var fetchFlows = flagSet.Int("flows", 0, "Fetch a number of Flows. Use a negative value to watch forever.")
```

Comment:
I think I agree with Shaun about the structure of this CLI

e.g.,

```
calico-node --flows --n 100
```

Really we should revisit all of this and use a proper CLI framework to build this binary, but that's for another time!